deployment/config/testnet/contracts.json

```diff
@@ -77,7 +77,7 @@
       "chainId": 10002,
       "address": "0xDc74c34F88d17895e31792439eaCE7cB1ed17d3a",
       "constructorArgs": [
-        "0xa737b3feB1Dc4d1630647E54954564FB3c903f0A",
+        "0x81A35AdC868aa0c607Fa98203fb2eA3015854d97",
```

Comment:
Since this wasn't a redeployment, the constructor arguments for proxies shouldn't change.